decidim-elections/config/locales/en.yml

```diff
@@ -1210,7 +1210,8 @@ en:
         in_person_votes:
           complete_voting:
             available_answers: 'Available answers:'
-            census_verified: The participant has not voted yet.
+            census_verified: The participant has not voted in person yet.
+            census_verified_with_online_vote: The participant has not voted in person, but has voted online. An offline vote will spoil any online vote.
```

Comment:
```suggestion
            census_verified_with_online_vote: This participant has already voted online. If they vote in person, the previous votes will be invalidated and this will be the definitive vote.
```